.github/workflows/release.yml

```diff
@@ -4,18 +4,19 @@ name: Check and realese cosmpy
 # if package version is newer it build, upload, tag and make a release page on github
 # secrets PYPI_USERNAME and PYPI_PASSWORD are required!
 on:
-  push:
-    branches: [ master ]
+  pull_request:
```

Comment:
Oh actually this needs to remain `push` and not `pull_request`. My mistake. Please revert.